core/tests/manual/main.rs

```diff
@@ -0,0 +1,123 @@
+use std::{fs::File, io::read_to_string};
+
+use comrak::{
+    arena_tree::{Node, NodeEdge},
+    nodes::{Ast, AstNode, NodeCodeBlock, NodeValue},
+    parse_document, ComrakOptions,
+};
+use nickel_lang_utils::{project_root::project_root, test_program};
+use test_generator::test_resources;
+use typed_arena::Arena;
+
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+enum CodeBlockType {
+    /// A code block intended to be evaluated in its entirety
+    Single,
+    /// A code block containing multiple lines, each of which should be
+    /// evaluated separately
+    Lines,
+    /// A code block that should only be parsed, not evaluated
+    Parse,
+    /// A code block containing a REPL session, e.g.
+    /// ```nickel repl
```

Comment:
The example doesn't check out with practice: it seems that you are requiring a `-` instead of a space, ie `nickel-repl`.

Reply:
Right, I tried what you were suggesting about spaces before. Unfortunately, gatsby doesn't seem to ignore the extra information after `nickel`. In fact, it seems to remove the space after `nickel` and try to look for a Prismjs language `nickelrepl` if given `nickel repl` in the code block. Relying on that behaviour felt a bit too spooky for me, though...

Reply:
On the other hand, relying on it and just continuing to check when updating the website should make syntax highlighting work in saner markdown renderers, I guess. It's easy to change back here, so whichever you think would make more sense long term.

Reply:
Yeah, and what about using a special character and a space, something like `nickel #repl` or `nickel !repl`? Gatsby would parse that as `nickel#repl` which has almost no chance of conflicting with anything, keeping the alias approach (could even be `nickel -repl`, even if that feels...slightly wrong for some reason), and such that other markdown parser would be ok with it? I tried it into Neovim and some random HTML markdown renderer extension I have and indeed adding a space preserves highlighting, while the dash doesn't. It seems like it doesn't make a difference for Gatsby in one direction or the other, but preserve other renderers

Reply:
I like `nickel #...` 👍 I'll make the change here and update the gatsby config on the website.